ESMF/GOCART2G_GridComp/DU2G_GridComp/DU2G_GridCompMod.F90

```diff
@@ -38,15 +38,27 @@ module DU2G_GridCompMod
 
    integer, parameter         :: NHRES = 6
 
+!  !Supported dust schemes
+   enum, bind(C)
+     enumerator :: DUST_SCHEME_DATA    = 0
+     enumerator :: DUST_SCHEME_GOCART
+     enumerator :: DUST_SCHEME_FENGSHA
+   end enum
+
 !  !Dust state
    type, extends(GA_GridComp) :: DU2G_GridComp
-       real, allocatable      :: rlow(:)        ! particle effective radius lower bound [um]
-       real, allocatable      :: rup(:)         ! particle effective radius upper bound [um]
+       real, allocatable      :: rlow(:)        ! particle radius lower bound [um]
+       real, allocatable      :: rup(:)         ! particle radius upper bound [um]
        real, allocatable      :: sfrac(:)       ! fraction of total source
+       real, allocatable      :: sdist(:)       ! aerosol fractional size distribution [1]
+       real                   :: alpha          ! FENGSHA scaling factor
+       real                   :: gamma          ! FENGSHA tuning exponent
+       real                   :: kvhmax         ! FENGSHA max. vertical/horizontal mass flux ratio [1]
        real                   :: Ch_DU_res(NHRES) ! resolutions used for Ch_DU
        real                   :: Ch_DU          ! dust emission tuning coefficient [kg s2 m-5].
        logical                :: maringFlag=.false.  ! maring settling velocity correction
        integer                :: day_save = -1      
+       integer                :: emission_scheme     ! emission scheme selector
```

Comment:
I have a modest preference to use strings rather than magic numbers for these.   Named constants are ok inside the code but are opaque in the resource file.

Reply:
Commit 261a4ac7a4d08fd0cee7050ed3c1b2f9817bd235 implements string options for the emission schemes.

Available options at this time are:
- Ginoux (original GOCART scheme)
- FENGSHA (NOAA/ARL scheme)
```
# Emissions methods
emission_scheme: Ginoux       # available schemes: Ginoux, FENGSHA
```